### PR TITLE
[15.0][FIX] web_responsive kanban mode for popup in Many2One fields on mobile

### LINKED ISSUE #2214: [15.0] web_responsive: Use search more... dialog in responsive views on m2x widget
I use Odoo 15 CE and web_resposive module

I hope the fields in list view can show full length of the field text content

Screenshot_2022-05-10-23-39-24-66_e4424258c8b8649f6e67d283a50a2cbc

### PR DESCRIPTION
**Pull request by SplashS**
This is enhancement for mobile mode.
When user clicks on Many2One field on mobile popup is shown.
Now the popup has list mode. On small mobile there are doubts to read data in list.
This PR switches this popup to kanban mode to give ability to read data more comfortably.

This closes #2214

### CODE REVIEW

**Comment by SplashS** [comment]
There is problem with css so this is still WIP

**Comment by SplashS** [comment]
Problem was solved.
@Tardo, could you review please.

**Review by pedrobaeza — APPROVED** [review]
Functionally tested on runboat

File: web_responsive/static/src/legacy/js/web_responsive.js [hunk 1]
```diff
@@ -169,6 +169,7 @@ odoo.define("web_responsive", function (require) {
     ViewDialogs.SelectCreateDialog.include({
         init: function () {
             this._super.apply(this, arguments);
+            this.viewType = "kanban";
```
Comment (Tardo) [inline web_responsive/static/src/legacy/js/web_responsive.js]: This should only happen on mobile phones
```suggestion
            if (config.device.isMobile) {
                    this.viewType = "kanban";
            }
```

**Review by Tardo — CHANGES_REQUESTED** [review]
There are some defects (not related to this PR):

- With web_responsive:
kanban_oca

- Without web_responsive:
kanban_odoo_ce

File: web_responsive/static/src/legacy/js/web_responsive.js (hunk 1 above)
Comment (SplashS) [reply web_responsive/static/src/legacy/js/web_responsive.js]: > There are some defects (not related to this PR)

Thank you! I've found problem with redundant padding on kanban card. But I can't reproduce situation with controls on the first screenshot. Could you please describe steps to reproduce?

> This should only happen on mobile phones

Shame on me. Fixed.

**Review by Tardo — APPROVED** [review]
Thanks! 

P.S: I will open a pr to fix the pagination problem as soon as I can...

**Pull request merged**